anta/cli/check/utils.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/python
```

Comment:
https://stackoverflow.com/questions/6908143/should-i-put-shebang-in-python-scripts-and-what-form-should-it-take 

I would rather we don't include it as we don't really need the file to be executable but better to use `/usr/bin/env python`